db/schema.rb

```diff
@@ -43,8 +43,6 @@
     t.boolean  "fogged"
   end
 
-  add_index "checkins", ["device_id"], name: "index_checkins_on_device_id", using: :btree
```

Comment:
@limzykenneth not sure why it's not working as the migration file for redirect_url is in your branch but shouldn't be an issue as long as we migrate dev and the schema isn't changed before it goes into master. Will merge this request now.